README.md

```diff
@@ -108,21 +108,38 @@ optional arguments:
 
 ### Graphic
 
-The `graphic` sub-command will create either a comprehensive diagram showing ontology modules together with classes, object properties and "Things" together with the path of imports, or (if the 'wee' option is selected) a simple diagram of ontology modules and the import hierarchy.  Graphics are exported both as ```png``` files and also as a ```dot``` file.  This ```dot``` file can be used with Graphviz or with web tools such as [Model Viewer](http://www.semantechs.co.uk/model-viewer)
+The `graphic` sub-command will create either 
+* a comprehensive diagram showing ontology modules together with classes, object properties and "Things"
+  together with the path of imports, or (if the 'wee' option is selected) a simple diagram of ontology modules and
+  the import hierarchy, or
+* a diagram of the use of classes and object and data properties in a triple store.
+    
+Graphics are exported both as ```png``` files and also as a ```dot``` file.  This ```dot``` file can be used with Graphviz or with web tools such as [Dot Viewer](http://www.semantechs.co.uk/turtle-editor-viewer/)
 
 ```
-$ onto_tool graphic -h
-usage: onto_tool graphic [-h] [--debug] [-o OUTPUT] [-v VERSION] [-w]
+usage: onto_tool graphic [-h] [-e ENDPOINT] [--schema | --data] [--debug]
+                         [-o OUTPUT] [--instance-limit INSTANCE_LIMIT]
+                         [--predicate-threshold PREDICATE_THRESHOLD]
+                         [-v VERSION] [-w]
                          [ontology [ontology ...]]
 
 positional arguments:
   ontology              Ontology file, directory or name pattern
 
 optional arguments:
   -h, --help            show this help message and exit
+  -e ENDPOINT, --endpoint ENDPOINT
+                        URI of SPARQL endpoint to use to gather data
+  --schema              Generate ontology import graph (default)
+  --data                Analyze instances for types and links
   --debug               Emit verbose debug output
   -o OUTPUT, --output OUTPUT
                         Output directory for generated graphics
+  --instance-limit INSTANCE_LIMIT
+                        Size limit on instance queries (default 500000)
```

Comment:
Suggest replace
- Size limit on instance queries (default 500000)

with

- Specify a limit on how many triples to consider that use any one predicate to find (default 500000). This option may result in an incomplete version of the diagram, missing certain links.


Or maybe I just have no idea what this means. What is an 'instance query'? Don't you mean using the --data option?